app/(root)/(case)/case-detail/page.tsx

```
@@ -115,6 +114,11 @@ function CaseDetailContent() {
           break;
         }
 
+        if (!!workflowIdMsg) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)